torch/_dynamo/convert_frame.py

```diff
@@ -793,7 +798,12 @@ def log_bytecode(
                 # We now have a new "last attempt", reset the clock
                 last_attempt_start_time = time.time()
                 if attempt > 100:
-                    unimplemented("100+ RestartAnalysis() calls")
+                    unimplemented_v2(
+                        gb_type="Excessive RestartAnalysis() calls",
+                        context="",
+                        explanation="Dynamo attempted to restart analysis 100+ times.",
```

Comment:
Some more information here about why this is bad would be good, like "we're recompiling too much, you are probably getting perf hits, so we decided stop compiling"